tests/test_utils/test_schemaview.py

```diff
@@ -353,6 +353,21 @@ def test_caching(self):
         view.add_class(ClassDefinition('W'))
         self.assertCountEqual(['Y', 'Z', 'W'], view.all_classes())
 
+    def test_import_map(self):
+        """
+        Path to import file should be configurable
+        """
+        for im in [{"core": "/no/such/file"}, {"linkml:": "/no/such/file"}]:
+            with self.assertRaises(FileNotFoundError):
+                view = SchemaView(SCHEMA_WITH_IMPORTS, importmap=im)
+                view.all_classes()
```

Comment:
If the _importmap_ does not have existent paths to YAML files, then wouldn't it be better if the program errored out in the schema initialization step itself?

As in should we be waiting till the user makes a call to one of the SchemaView methods (in this case `all_classes()`) before they realize that they've provided an incorrect importmap?

```python
view = SchemaView(SCHEMA_WITH_IMPORTS, importmap=im)
```
itself should complain about non existent files.